Makefile

```
@@ -39,11 +39,18 @@ ifdef CUSTOM_CA
 	cp -f ${CUSTOM_CA} docker/conf/certificates/
 endif
 
+up-standalone: update-box custom_ca
+ifeq ($(GITHUB_ACTIONS),true) # Always set to true when GitHub Actions is running the workflow. You can use this variable to differentiate when tests are being run locally or by GitHub Actions.
+	SSL_CERT_FILE=${SSL_CERT_FILE} CURL_CA_BUNDLE=${CURL_CA_BUNDLE} ANSIBLE_ARGS='--extra-vars "ci_test=true cluster_mode=false"' vagrant up --provision
+else
+	SSL_CERT_FILE=${SSL_CERT_FILE} CURL_CA_BUNDLE=${CURL_CA_BUNDLE} CUSTOM_CA=${CUSTOM_CA} ANSIBLE_ARGS='--extra-vars "cluster_mode=false"' vagrant up --provision
```

I would recommend using the same names that we use in variables.tf, mode=`standalone` or `cluster`

```suggestion
	SSL_CERT_FILE=${SSL_CERT_FILE} CURL_CA_BUNDLE=${CURL_CA_BUNDLE} CUSTOM_CA=${CUSTOM_CA} ANSIBLE_ARGS='--extra-vars "mode=standalone"' vagrant up --provision
```

---

Fair enough. I used a bool because there is less ambiguity in that, and it'll automatically throw an error if it is not defined properly. What do you think, use the bool `cluster_mode` or the string `mode`?

---

I think `bool` is restricted by having switch only between 2 cases, but the string is multi-switch.

U can also create folders for each example under `dev/ansible/`, leave `playbook.yml` there, and implement that switch, to load tasks from the proper folder. It could be useful, due that some tasks won't be passing to all examples.

---

Ah, that is a good point, sounds like a more flexible solution to me. I'll get on it

---

Although maybe we should actually implement a generic solution to this problem. I imagine several examples will be a problem in more than just presto. What about just implementing this "hotfix" for now, and start working on a more permanent solution in the template?

---

I've updated to use the `mode` variable, ok for you if I close?https://github.com/fredrikhgrelland/terraform-nomad-presto/blob/c10eba8da46240295ddf849bb0497d0c02cc55d5/Makefile#L49-L54